accounts_from_snapshot/examples/import_archive.rs

```diff
@@ -0,0 +1,43 @@
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::time::Instant;
+use log::{info, trace};
+use lite_accounts_from_snapshot::import::import_archive;
+
+#[tokio::main]
+pub async fn main() {
+    tracing_subscriber::fmt::init();
+
+    let snapshot_file = "/Users/stefan/mango/projects/accountsdb-how-it-works/snapshot-278240833-2WmWBr6Fnq9w6VVtLbhezGZHfH6L7YrPgggRqFg2ouTk.tar.zst";
+
+    let (mut accounts_rx, _) = import_archive(PathBuf::from_str(snapshot_file).unwrap()).await;
```

Comment:
(nit) use path vom argv and env_logger, like all other examples:
```suggestion
    env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info")).init();

    let snapshot_file = std::env::args_os()
        .skip(1)
        .next()
        .expect("snapshot file path is required");

    let (mut accounts_rx, _) = import_archive(snapshot_file.into()).await;
```